astropy_healpix/core.py

```diff
@@ -77,10 +84,92 @@ def level_to_nside(level):
     nside : int
         The number of pixels on the side of one of the 12 'top-level' HEALPix tiles.
     """
+    level = np.asarray(level, dtype=np.int64)
+
     _validate_level(level)
     return 2 ** level
 
 
+def nside_to_level(nside):
```

Comment:
Should we expose this as part of the public API?

I guess "yes", even if it's just one line plus a type cast and input validation, because it leads to more readable code than calling `np.log2` all over the place directly?

But I'm not sure: @astrofrog and @lpsinger - please comment.

Reply:
Yes please!